.local.env

```diff
@@ -1,9 +1,14 @@
 NODE_ENV=development
 
 API_CORE_BASE=https://cube-creator.lndo.site/
+VUE_APP_API_URL=https://cube-creator.lndo.site/
 
 UI_BASE=https://app.cube-creator.lndo.site/
 
 STORE_QUERY_ENDPOINT=http://store:3030/cube-creator/query
 STORE_UPDATE_ENDPOINT=http://store:3030/cube-creator/update
 STORE_GRAPH_ENDPOINT=http://store:3030/cube-creator/data
+
+AUTH_AUDIENCE=https://datacube.zazukoians.org/
+AUTH_ISSUER=https://keycloak.zazukoians.org/realms/zazuko-dev
+AUTH_CLIENT_ID=cube-creator
```

Comment:
Not sure we should have this in the repo...

Reply:
I just saw #5. We're just gonna have to merge some stuff ;)

Reply:
Maybe you'd merge this into #5 and we handle OIDC in one PR?